Exiled.Events/Patches/Generic/StaminaUsagePatch.cs

```diff
@@ -0,0 +1,27 @@
+// -----------------------------------------------------------------------
+// <copyright file="StaminaUsagePatch.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Generic
+{
+    using Exiled.API.Features;
+
+#pragma warning disable SA1313
+
+    using HarmonyLib;
+
+    /// <summary>
+    /// Patches <see cref="Stamina.ProcessStamina()"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(Stamina), nameof(Stamina.ProcessStamina))]
+    internal class StaminaUsagePatch
+    {
+        private static bool Prefix(Stamina __instance)
+        {
+            return Player.Get(__instance._hub).IsUsingStamina;
```

Comment:
While this probably won't be an issue, you should do
```
return Player.Get(__instance._hub)?.IsUsingStamina``` since Player.Get() can return null.

Reply:
Fixed